From: Patrycja Rosa <alpine@ptrcnull.me>
Date: Sat, 12 Feb 2022 18:36:03 +0100
Subject: increase performance test timeout

ppc64le seems to have issues with finishing in 3 seconds,
increasing this timeout allows for successful builds on that architecture

diff --git a/test_zipp.py b/test_zipp.py
--- a/test_zipp.py
+++ b/test_zipp.py
@@ -297,1 +297,1 @@
-    @func_timeout.func_set_timeout(3)
+    @func_timeout.func_set_timeout(10)
